packages/yourloops/test/lib/team/team-api.test.tsx

```diff
@@ -27,9 +27,128 @@
+
+  describe('getTeams', () => {
+    it('should get a list a teams', async () => {
+      const data: ITeam[] = [
+        { name: 'team1' } as ITeam,
+        { name: 'team2' } as ITeam
+      ]
+      jest.spyOn(HttpService, 'get').mockResolvedValueOnce({ data } as AxiosResponse)
+
+      const teams = await TeamApi.getTeams()
+      expect(teams).toEqual(data)
+      expect(HttpService.get).toHaveBeenCalledWith({ url: '/v0/my-teams' })
+    })
+  })
+
+  describe('getPatients', () => {
+    it('should get a list a patients', async () => {
+      const data: ITeamMember[] = [
+        { email: 'member1@team.com' } as ITeamMember,
+        { email: 'member2@team.com' } as ITeamMember
+      ]
+      jest.spyOn(HttpService, 'get').mockResolvedValueOnce({ data } as AxiosResponse)
+
+      const patients = await TeamApi.getPatients()
+      expect(patients).toEqual(data)
+      expect(HttpService.get).toHaveBeenCalledWith({ url: '/v0/my-patients' })
+    })
+  })
+
+  describe('invitePatient', () => {
+    it('should invite a new patient in a team and get a notification if success', async () => {
+      const data = { creatorId: 'creatorId' } as INotificationAPI
+      jest.spyOn(HttpService, 'post').mockResolvedValueOnce({ data } as AxiosResponse)
+
+      const notification = await TeamApi.invitePatient({ teamId, email })
+      expect(notification).toEqual(data)
+      expect(HttpService.post).toHaveBeenCalledWith({
+        url: '/confirm/send/team/invite',
+        payload: { teamId, email, role: UserRoles.patient },
+        config: { headers: { [HttpHeaderKeys.language]: getCurrentLang() } }
+      })
+    })
+  })
+
+  describe('inviteMember', () => {
+    it('should invite a new member in a team and get a notification if success', async () => {
+      const data = { creatorId: 'creatorId' } as INotificationAPI
+      jest.spyOn(HttpService, 'post').mockResolvedValueOnce({ data } as AxiosResponse)
+
+      const notification = await TeamApi.inviteMember({ teamId, email, role })
+      expect(notification).toEqual(data)
+      expect(HttpService.post).toHaveBeenCalledWith({
+        url: '/confirm/send/team/invite',
+        payload: { teamId, email, role },
+        config: { headers: { [HttpHeaderKeys.language]: getCurrentLang() } }
+      })
+    })
+  })
+
+  describe('createTeam', () => {
+    const name = 'super team'
+    const phone = 'iPhone 5'
+    const address = { city: 'grenoble' } as PostalAddress
+    const newTeam = { name, phone, address } as ITeam
+
+    it('should create a new team', async () => {
+      jest.spyOn(HttpService, 'post').mockResolvedValueOnce({ data: newTeam } as AxiosResponse)
+      const team = await TeamApi.createTeam(newTeam)
+      expect(team).toEqual(newTeam)
+      expect(HttpService.post).toHaveBeenCalledWith({
+        url: '/crew/v0/teams',
+        payload: { ...newTeam, type: TeamType.medical }
+      })
+    })
+
+    it('should failed if missing mandatory parameters', async () => {
```

Comment:
I suggest having two more test like this one (one with phone missing and one with address missing)